spec/unit/puppet/provider/registry_value_spec.rb

```diff
@@ -93,6 +93,7 @@ def create_and_destroy(path, reg_type, data)
 
       reg_value.provider.create
       reg_value.provider.exists?.should be_true
+      expect(reg_value.provider.data).to eq([data].flatten)
```

Comment:
We do not explicitly have such tests.

However, as long as the randomly generated values here are above 0xFF, we automatically test endian-ness as a side effect.  If it's wrong, we will have test failures here.

I also thought about adding acceptance tests that have hardcoded values that are known to trigger endianness flubs, but fixing the current manifest to do that instead of injecting "phase" is not currently straightforward.